packages/playground/src/App.js

```diff
@@ -140,8 +142,9 @@ export default class extends React.Component {
   }
 
   handleLanguageChange({ target: { value } }) {
-    this.setState(() => ({ language: value }), () => {
-      document.querySelector('html').setAttribute('lang', value || window.navigator.language);
+    const lang = value || window.navigator.language;
+    this.setState(() => ({ language: value, direction: this.getDirection(lang) }), () => {
```

Comment:
Sorting by name, and split into multiple lines.

```suggestion
    this.setState(() => ({
      direction: this.getDirection(lang),
      language: value
    }), () => {
```